### ISSUE TITLE
Logic - GetValidInput Output

### ISSUE THREAD

**Issue by jwood36** [TatumS1/HW_TicTacToe_Task1_-tstigna1-.md#20]
When the player prematurely quits the game, you say "Game Over" yet, they would never see the message.

```csharp
    static bool GetValidInput(out int choice) {
      choice = 0;
      var input = Console.ReadLine();
      if (input.ToLower() == "q") {
        Console.WriteLine("G A M E  O V E R");
        Environment.Exit(0);
      }
      if (!int.TryParse(input, out choice) || choice < 1 || choice > 9) {
        return false;
      }
      return true;
    }

```

**Comment by TatumS1** [comment]
The message displays properly when I run this and end it prematurely

**Comment by TatumS1** [comment]
Now that I have resolved #22 I wonder if this issue would persist for you
